src/rtcModule/webrtc.cpp

```diff
@@ -2621,7 +2621,21 @@ void Call::onClientLeftCall(Id userid, uint32_t clientid)
         }
 
         SUB_LOG_DEBUG("ENDCALL received for ourselves, finishing the call");
-        destroy(TermCode::kErrNetSignalling, false);
+        auto wptr = weakHandle();
+        destroy(TermCode::kErrNetSignalling, false)
+        .then([wptr, this]
+        {
+            karere::Id chatid = chat().chatId();
+            auto it = mManager.mRetryCall.find(chatid);
+            if (it == mManager.mRetryCall.end())
+            {
+                karere::AvFlags flags = sentAv();
+                mManager.mRetryCall[chatid] = std::pair<karere::AvFlags, bool>(flags, true);
```

Comment:
You already looked for the `chatid` and it's available, simply use it.
```suggestion
                it->second = std::pair<karere::AvFlags, bool>(flags, true);
```

Reply:
Yes, I looked for it but I didn't find it, then I added it.
We retry the call if there isn't other retry on fly

Reply:
Ouch :facepalm: You're totally right, I misread the test.